python/cookiecutter-django.md

```diff
@@ -0,0 +1,538 @@
+    - The workflow name is the value of the first configuration `name`, in the example above is `Python application`.
+
+```markdown
+# Template:
+.. image:: https://github.com/<OWNER>/<REPOSITORY>/workflows/<WORKFLOW_NAME>/badge.svg
+
+# Example:
+.. image:: https://github.com/afmicc/cookiecutter_starter/workflows/Python%20application/badge.svg
+    :alt: Tests status
+```
+
+<details>
+  <summary>.github/workflows/python-app.yml. <i>(Click &#x25B6; to display the file)</i></summary>
+
+```yml
+#  .github/workflows/python-app.yml
+
+name: Python application
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    services:
+      db:
+        image: postgres:11.6-alpine
+        env:
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_USER: postgres
+        ports:
+          - "5432:5432"
+    env:
+      DATABASE_URL: postgresql://postgres:postgres@localhost:5432/postgres
+      CC_TEST_REPORTER_ID: ${{ secrets.CC_TEST_REPORTER_ID }}
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+
+    - name: Installing resoruces
+      run: |
+        pip install --upgrade pip==20.0.2
+        pip install pipenv
+
+    - name: Cache pipenv
+      uses: actions/cache@v2
+      id: cache-venv
+      with:
+        path: ./.venv
+        key: ${{ runner.os }}-venv-${{ hashFiles('Pipfile.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-venv-
+
+    - name: Installing requirements
+      run: |
+        pipenv install --dev
+
+    - name: Checking PEP8 code style
+      run: |
+        pipenv run flake8 --count
+
+    - name: Checking Black code formatter
+      run: |
+        pipenv run black . --check
+
+    - name: Running tests
+      run: |
+        pipenv run coverage run -m pytest --ds=config.settings.test
+
+    - name: Checking coverage
+      run: |
+        pipenv run coverage report --fail-under=90 -m
+        pipenv run coverage xml
+
+    - name: Setup Code Climate test-reporter
+      run: |
+          curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+          chmod +x ./cc-test-reporter
+          ./cc-test-reporter before-build
+          ./cc-test-reporter after-build --coverage-input-type coverage.py --exit-code $?
+```
+
+</details>
+
+## Adding a new app
+
+1. To create a new app into the project, we have to run the command `$ pipenv run python manage.py startapp <new_app_name>`.
+    - **new_app_name**: is the name of the new app and it has to be written in plural. For example: targets
+1. How that command creates the new app at the root directory, you have to move it to `<project_name>` directory, where there are other existing apps like `users`.
```

Comment:
```suggestion
1. Since that command creates the new app at the root directory, you have to move it to `<project_name>` directory, where there are other existing apps like `users`.
```